crafters/image/AlbumentationsCrafter/manifest.yml

```diff
@@ -0,0 +1,12 @@
+manifest_version: 1
+name: AlbumentationsCrafter
+kind: crafter
+description: Apply transformations from the Albumentations package to the image.
+author: Jina AI Dev-Team (dev-team@jina.ai)
```

Comment:
Hey @tadejsv , is not the root of the problem, so no big deal, but you are welcome to attribute yourself as the author of this Hub Executor